.github/workflows/npm-publish-github-packages.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow will run tests using node and then publish a package to GitHub Packages when a release is created
+# For more information see: https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages
+
+name: Node.js Package
+
+on:
```

Comment:
ok well this is just an attempt to steal an npm publish token, but it's kind of foolish on something that's not published to npm at all.